audio/file_contexts

```diff
@@ -0,0 +1 @@
+/vendor/bin/set_board_prop.sh u:object_r:set_board_prop_exec:s0
```

Comment:
Please ensure you are using "#!/vendor/bin/sh"

Reply:
thanks, script already ensured
https://github.com/projectceladon/device-androidia-mixins/pull/2191/files#diff-c2869d63e6d7b2d925bd61e8a4f791ae599064af30868e98374fdc73f8c07b22